install/installer/pkg/components/public-api-server/configmap.go

```diff
@@ -41,7 +42,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	})
 
 	cfg := config.Configuration{
-		GitpodServiceURL:                  fmt.Sprintf("wss://%s", ctx.Config.Domain),
+		GitpodServiceURL:                  fmt.Sprintf("ws://%s.%s.svc.cluster.local:%d", server.Component, ctx.Namespace, server.ContainerPort),
```

Comment:
I think you could simplify it to just `server.Component:server.ContainerPort` as they are in the same namespace

Reply:
I think this would work, but I'd like to leave it this way for consistency with how we find the usage service a couple of lines below. It also makes it very clear that it's a cluster-internal address.